src/NodaTime.Test/Testing/TimeZones/FakeDateTimeZoneSourceTest.cs

```diff
@@ -129,15 +129,17 @@ public void GetEnumerator()
         // Unfortuntely NUnit requires the exact exception type :(
         private static void AssertBuildFails(FakeDateTimeZoneSource.Builder builder)
         {
+            bool success = false;
             try
             {
                 builder.Build();
-                Assert.Fail("Expected exception");
```

Comment:
Nice fix. But how is this related to the NUnit upgrade? (I don't mind either way, just curious.)

Reply:
While we were still catching the exception in the latest NUnit, it was being reported as a failure - I suspect that there's some out-of-band way that it records errors, even if an exception isn't thrown.

Reply:
(I haven't dug into it though - that's just a guess.)

Reply:
Doh, and you even said as much in the commit message, which I did read.

Thanks for explaining (again!)